projects/command/src/command_pattern_implementation/concerns/result_type.rb

```diff
@@ -18,7 +18,11 @@ def result_type
           end
         end
 
-        foobara_delegate :result_type, :raw_result_type_declaration, to: :class
+        foobara_delegate :raw_result_type_declaration, to: :class
```

Comment:
raw_result_type_declaration no longer exists I don't think. So we can just delete that one.